plugins/Super_Saw/DistrhoPluginInfo.h

```diff
@@ -0,0 +1,11 @@
+#ifndef DISTRHO_PLUGIN_INFO_H_INCLUDED
+#define DISTRHO_PLUGIN_INFO_H_INCLUDED
+
+#define DISTRHO_PLUGIN_NAME  "Super Saw"
+#define DISTRHO_PLUGIN_URI   "https://github.com/cranixx/Super_Saw_DPF"
```

Comment:
This URI should be like the other plugins, if it is going to be part of DISTRHO examples.